src/main/java/com/mcmoddev/mmdbot/core/BotConfig.java

```diff
@@ -21,16 +21,47 @@
 
 /**
  * The configuration holder for the bot.
+ *
+ * @author
+ *
  */
 public final class BotConfig {
+
+	/**
+	 *
+	 */
+    private static final String COMMUNITY_CHANNEL_OWNER_PERMISSIONS = "community_channels.owner_permissions";
+
+    /**
+     *
+     */
     private final CommentedFileConfig config;
-    private boolean newlyGenerated = false;
 
-    public BotConfig(Path configFile) {
+    /**
+     *
+     */
+    private boolean newlyGenerated;
+
+    /**
+     *
+     */
+    private static final String COMMANDS = "commands.";
```

Comment:
Maybe rename this constant to `COMMANDS_PREFIX`, to clarify that it is a prefix for config options instead of a config option itself (like with `COMMUNITY_CHANNEL_OWNER_PERMISSIONS`)?

Reply:
Fixed.